packages/suite/src/constants/suite/routes.ts

```diff
@@ -72,13 +60,13 @@ const routes = [
         name: 'onboarding-index',
         pattern: '/onboarding',
         app: 'onboarding',
-        isModal: true,
-        params: modalAppParams,
+        isModal: true, // used to prevent url change
```

Comment:
nitpick, I'd find this more clear
```suggestion
        isModal: true, // Prevent user from manual URL twiddling.
```

the important difference being that the user is who we're trying to prevent from changing the url.

Reply:
Interesting I didn't even thought of that. I did it it more for browser than a user 😁  you can get to onboarding from suite modal when you connect initialised device, and for various reasons we don't want the browser to change the URL. I did not really have enough time to examine this further so I am not sure what are all the reasons, but one of them is that browser's back button would be active if there was another route on the history stack and we would need to disable it anyway cause you can't really go back in the middle of onboarding process.

Reply:
aha! Then this is more complicated than I thought! The original comment now makes more sense. Thanks. Resolving.